improver_tests/acceptance/test_weighted_blending.py

```diff
@@ -216,6 +200,7 @@ def test_cycletime_with_specified_frt(tmp_path):
     acc.compare(output_path, kgo_path)
 
 
+# TODO this should break!  Need to deal with single inputs
```

Comment:
Does this mean this needs to break in the future or it will break now?

Reply:
Ah no, it means I forgot to remove the TODO when I fixed the code!  I'll remove this.